recipes/ledger/meta.yaml

```
@@ -0,0 +1,61 @@
+{% set name = "ledger" %}
+{% set version = "3.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ledger/ledger/archive/v{{ version }}.tar.gz
+  sha256: b248c91d65c7a101b9d6226025f2b4bf3dabe94c0c49ab6d51ce84a22a39622b
+
+build:
+  number: 0
+  skip: True  # [not (py<=27 and unix)]
```

The python version is already specified via the requirements, so this can be simplified to:
```suggestion
  skip: True  # [win]
```